string name
bool value
---
industrial_msgs/ServiceReturnCode code